tests/chainer_tests/link_hooks_tests/test_spectral_normalization.py

```diff
@@ -1,14 +1,20 @@
+import copy
+import os
+import shutil
+import tempfile
 import unittest
 
 import numpy
 import pytest
 
 import chainer
-from chainer.backends import cuda
+from chainer.backends._cpu import _to_cpu as to_cpu
```

Comment:
Could you import only modules?
`from chainer.backends import _cpu` or `import chainer.backends._cpu as cpu_backend`, etc.

Reply:
Sure.